invenio_search/views/search.py

```diff
@@ -257,8 +251,6 @@ def search(collection, p, of, ot, so, sf, sp, rm, rg, jrec):
         filtered_facets=filtered_facets,
         response=response,
         rg=rg,
-        create_nearest_terms_box=lambda: _("Try to modify the query."),
```

Comment:
This is safe to remove because we are not using it in the templates right?

```
$ ag create_nearest_terms_box
invenio-search/invenio_search/templates/search/browse_base.html
78:    {{ create_nearest_terms_box()|safe }}

invenio-search/invenio_search/templates/search/search_base.html
63:            {{ create_nearest_terms_box()|safe }}

invenio-search/invenio_search/views/search.py
264:        create_nearest_terms_box=lambda: _("Try to modify the query."),

invenio-utils/invenio_utils/url.py
562:       Used to be used for search engine's create_nearest_terms_box,

```